### ISSUE TITLE
Web View/Remote Control

### ISSUE THREAD

**Issue by cadab** [tomahawk-player/tomahawk#463]
Hi,

Ive had a look around but cant see anything documented. I'd love to be able to use this app as a music media centre, is it possible to add songs/edit playlists via a remote control, e.g. from a phone/tablet.

**Comment by dschmidt** [comment]
There is a remote web api, but I'm afraid it's nowhere near as powerful as you would like it to be :-(

Currently it only supports prev/next/pause/play and getting basic information about the currenlty playing track.
Compare https://github.com/tomahawk-player/tomahawk/blob/master/src/libtomahawk-playdarapi/Api_v1_5.cpp#L52
and see what requests happen here: http://localhost:60210/ (e.g. http://localhost:60210/api/1.5/playback/playpause)

**Comment by e13mort** [comment]
Hi @dschmidt 
I'm able to see the web api at localhost, but can't access it over my local network. Is that a player bug or maybe a network configuration issue? 
Thank you.

**Comment by muesli** [comment]
@e13mort Have you enabled "Allow other computers to interact with Tomahawk" in the advanced settings? Otherwise Tomahawk will only listen for connections coming from localhost.

**Comment by e13mort** [comment]
@muesli No, I haven't. Thank you. 

BTW, have you thought about a feature which allows to control a desktop player from the android version?

**Comment by muesli** [comment]
Yeah, we have talked about that before and definitely want to integrate that sooner or later.
